test/integration/conditions.test.ts

```diff
@@ -37,6 +37,13 @@ describe('conditions schema', () => {
     expect(result.error).toEqual(undefined);
     expect(result.value.chain).toEqual(5);
   });
+
+  it('should validate chain id', async () => {
+    result = condition.validate({ chain: 10 });
```

Comment:
We should contain all variable declarations such as `result` to the `it(...)` closure where they are being used. This prevents state-leaking across different tests.